project/src/com/google/daggerquery/executor/models/Query.java

```diff
@@ -308,12 +302,50 @@ private void checkNodeForCorrectness(String node, BindingGraph bindingGraph) {
     List<String> closestNodes = new ArrayList<>();
 
     for (String node: allNodes) {
-      int distance = levenshteinDistanceCalculator.apply(originalNode, node);
+      int distance = calculateDistance(originalNode, node);
       if (distance <= MAX_NUMBER_OF_MISPLACED_LETTERS) {
         closestNodes.add(node);
       }
     }
 
     return closestNodes;
   }
+
+  /**
+   * Measures the difference between two strings.
```

Comment:
nit: Make it clear it's the Levenshtein distance in the Javadoc?

Reply:
👌